src/python/openvino_genai/py_openvino_genai.pyi

```diff
@@ -1706,6 +1706,21 @@ class Text2ImagePipeline:
         ...
     def set_scheduler(self, scheduler: Scheduler) -> None:
         ...
+class TextStreamer:
+    """
+    
+    TextStreamer is used to decode tokens into text and call a user-defined callback function.
+    
+    tokenizer: Tokenizer object to decode tokens into text.
+    callback: User-defined callback function to process the decoded text, callback should return either boolean flag or StreamingStatus.
+    
+    """
+    def __init__(self, tokenizer: ..., callback: typing.Callable[[str], bool | StreamingStatus]) -> None:
+        ...
+    def end(self) -> None:
+        ...
+    def write(self, arg0: int) -> StreamingStatus:
```

Comment:
please, use explicit argument name via `py::arg` to avoid automatic `arg0` names

Reply:
done